storybook/storybook-css/src/components/Breadcrumb/Breadcrumb.docs.mdx

```diff
@@ -0,0 +1,12 @@
+import { Markdown, Meta, Primary } from "@storybook/blocks";
+import * as Breadcrumb from "./Breadcrumb.stories.tsx";
+import README from "../../../../../packages/css/src/breadcrumb/README.md?raw";
+import css from "../../../../../packages/css/src/breadcrumb/breadcrumb-css.md?raw";
```

Comment:
We don't need this if it's empty, right?

Reply:
removed it.